application/actions/auth.go

```diff
@@ -262,9 +289,9 @@ func createAuthUser(
 		return AuthUser{}, err
 	}
 
-	var uos []AuthOrgOption
+	var uos []authOption
 	for _, uo := range user.Organizations {
-		uos = append(uos, AuthOrgOption{
+		uos = append(uos, authOption{
```

Comment:
Not sure what this does. I don't see any later access of `uos`. Does `authUser` need this list?

Reply:
It must be left-over code from when we were trying to prepare for Users potentially having more than one org.

Reply:
I'll remove it.